ui/src/main/webapp/src/app/registered-application/register-application-form.component.ts

```diff
@@ -60,7 +59,7 @@ export class RegisterApplicationFormComponent extends FormComponent implements O
         super();
 
         this.multipartUploader = _registeredApplicationService.getMultipartUploader();
-        this.multipartUploader.onSuccessItem = () => this.countUploadedApplications++;
+        this.multipartUploader.onSuccessItem = () => console.log("File uploaded");
```

Comment:
@jsight do you want the `console.log` instructions to be in the committed code?

Reply:
Yeah, I was just maintaining a placeholder. We can remove it later if need be.